modules/check_stop_condition.py

```diff
@@ -0,0 +1,60 @@
+"""
+Checks whether the drone has reached its max flight time and sends it back to launch.
+"""
+import dronekit
```

Comment:
Add empty line above this line. `dronekit` is a 3rd party module, so there is an empty line. `sys` and `time` are system modules, so there is no empty line.